### DISCUSSION TITLE
为什么我感觉v5没有v4的好看了

### DISCUSSION THREAD

**Discussion by caihai123** [ant-design/ant-design#41196] [General]
比如button和input的圆角，总感觉没之前的好看

**Comment by MadCcc** [comment]
可以通过动态主题配置哦，ref: https://ant-design.antgroup.com/docs/react/customize-theme-cn

**Reply by caihai123** [reply]
是的，这个已经看见了。另外能不能再请教一个问题，这个暗黑主题模式怎么开啊，我使用了theme.darkAlgorithm但是没什么效果

**Reply by caihai123** [reply]
我自己的问题，我把theme.darkAlgorithm设置到token里面去了😒
